satellite/repair/checker/checker.go

```diff
@@ -221,18 +182,97 @@ func (checker *Checker) updateSegmentStatus(ctx context.Context, pointer *pb.Poi
 		// we need one additional piece for error correction. If only the minimum is remaining the file can't be repaired and is lost.
 		// except for the case when minimum and repair thresholds are the same (a case usually seen during testing)
 	} else if numHealthy <= redundancy.MinReq && numHealthy < redundancy.RepairThreshold {
+
+		// make an entry in to the irreparable table
+		segmentInfo := &pb.IrreparableSegment{
+			Path:               []byte(path),
+			SegmentDetail:      pointer,
+			LostPieces:         int32(len(missingPieces)),
+			LastRepairAttempt:  time.Now().Unix(),
+			RepairAttemptCount: int64(1),
+		}
+
+		// add the entry if new or update attempt count if already exists
+		err := checker.irrdb.IncrementRepairAttempts(ctx, segmentInfo)
+		if err != nil {
+			return Error.New("error handling irreparable segment to queue %s", err)
```

Comment:
Same problem here with the `err` stack, use `Combine` or log `err`

Reply:
I will use Combine here